master

werpu opened the pull request:
Adding -joynum parameter which allows to set the daphne 
default joystick to another value than /dev/input/js0

usage -joynum 1 sets the joystick to /dev/input/js1

werpu committed:
Fix for joystick is only used from dev/input/js0

Shmoopty commented:
Thank you for this thoughtful contribution!

Can you review how your branch is mixing tabs and spaces?

GitHub is showing your commits as doing unusual changes in indentation.  You may need another commit to make the code consistent.

werpu commented:
sure I will have a look

werpu committed:
Fixing the indentations, the project
uses tabs (did not noptice that upfront)
added a small comment regarding the new parameter as well

werpu commented:
Fixed the intentations, I overlooked that you were using tabs and my ide was set to spaces.
Should be well now.
